src/Controller/Admin/ActivityController.php

```diff
@@ -107,6 +130,7 @@ public function newAction(Request $request): Response
         if ($form->isSubmitted() && $form->isValid()) {
             $location = $activity->getLocation();
             assert(null !== $location);
+            $activity->setIsArchived(false);
```

Comment:
I feel that, since this is a default, it should be set in the constructor of `Activity` instead.